pkg/cli/webhook/gitlab.go

```diff
@@ -90,7 +90,7 @@ func (gl *gitLabConfig) askGLWebhookConfig(controllerURL, apiURL string) error {
 		return err
 	}
 
-	if apiURL == "" && gl.Hosted {
+	if apiURL == "" && !strings.HasPrefix(repoURL, "https://gitlab.com") {
```

Comment:
most instance on gitlab are "hosted" compared to github where github.com is very much canonical, i think we should just ask the URL all the time